.buildkite/pipeline.yml

```diff
@@ -144,15 +144,15 @@ steps:
       - './nix/regenerate.sh'
     agents:
       system: ${macos}
-    if: 0 == 1 # Disabled for now until a macos mini is available
+    # if: 0 == 1 # Disabled for now until a macos mini is available
```

Comment:
Any reason to leave it commented out and not deleting?

Reply:
Yes. — It's temporary until will get dedicated macOS Agents.

Reply:
Do you mean that once we have those agents you'll un-comment the `if: 0 == 1`  back?

Reply:
> Do you mean that once we have those agents you'll un-comment the if: 0 == 1 back?

No, the other way round — once we have those Agents, I will delete the line.